tools/webdriver/webdriver/bidi/modules/input.py

```diff
@@ -291,30 +311,36 @@ def pointer_move(self,
                      duration: Optional[int] = None,
                      origin: Union[str, Mapping[str, Any]] = "viewport",
                      **kwargs: Any) -> "PointerInputSource":
-        self.actions.append(PointerMoveAction(x, y, duration=duration, origin=origin, **kwargs))
+        self.actions.append(
+            PointerMoveAction(x, y, duration=duration, origin=origin,
+                              **kwargs))
         return self
 
 
 class WheelInputSource(InputSource):
     input_type = "wheel"
 
-    def scroll(self,
-               x: int,
-               y: int,
-               delta_x: int = 0,
-               delta_y: int = 0,
-               duration: Optional[int] = None,
-               origin: Union[str, Mapping[str, Any]] = "viewport") -> "WheelInputSource":
-        self.actions.append(WheelScrollAction(x,
-                                              y,
-                                              delta_x=delta_x,
-                                              delta_y=delta_y,
-                                              duration=duration,
-                                              origin=origin))
+    def scroll(
+        self,
```

Comment:
Why the different indentation here compared to the others above? Is that what mypy suggests? Maybe because of the `origin` line being too long? I mean I like the 4 char indentation and not the 8 char one, but would like to know why it's different.